src/main/java/com/box/sdk/BoxAPIRequest.java

```diff
@@ -51,6 +51,18 @@
 
     /**
      * Constructs an unauthenticated BoxAPIRequest.
+     *
+     * @param url
+     *            the {@link URL} of the request
+     * @param method
+     *            the {@link HttpMethod} of the request
+     */
+    public BoxAPIRequest(URL url, HttpMethod method) {
```

Comment:
Yes, it can be a part of feedback process, sorry I am only hate String constants, where enum or codebook can be used. It was impulsive reaction, if you want String, I can revert it.